File: Math/Prime/Kotlin/is_prime.kt
package main

fun main(args: Array<String>) {  
   	var num = 9
    var isPrime = checkPrimeNumber(num)
    if (isPrime)
        println("$num is a prime number.")
    else
        println("$num is not a prime number.")
}

fun checkPrimeNumber(num: Int): Boolean {
    var flag = true
    if (num == 1) return false
    for (i in 2..num / 2) {
        if (num % i == 0) {
            flag = false
            break
        }
    }

    return flag
}

File: Math/Prime/isPrime.cpp
#include <bits/stdc++.h>

void isPrime(int a){
	if (x == 0 || x == 1) {
		cout<<"This number is not prime !\n";
		return;
	}
	int f = 1;
	for (int i = 2; i <= sqrt(x); i++)
		if (x % i == 0) f = 0;
	if (f) cout<<"This number is prime !\n";
	else cout<<"This number is not prime !\n";
}

int main() {
	int x;
	cin>>x;
	isPrime(x);
	return 0;
}
